scripts/grpc-integration-test.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+PS4='T$(date "+%H:%M:%S") '
+set -euxf -o pipefail
+
+# use global variables to reflect status of db
+db_is_up=
+
+usage() {
+  echo $"Usage: $0 <image_version>"
+  exit 1
+}
+
+check_arg() {
+  if [ ! $# -eq 1 ]; then
+    echo "ERROR: need exactly one argument, <image_version>"
```

Comment:
It seems like you're trying to run already published image of jaeger-remote-storage? We don't have to fix it in this pr but the preferred way is to run with the code built from THIS version of the repo, not some other version.

Reply:
Please add a ticket